<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputKey</class>
 <widget class="QWidget" name="InputKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Key_L">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_G">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_Enter">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>119</y>
     <width>60</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_J">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>J</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_O">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_F">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_K">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_Backspace">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>退格
←</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_D">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_Z">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_M">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_A">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_Minus">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_Period">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_X">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_Y">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_H">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_C">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_Space">
   <property name="geometry">
    <rect>
     <x>479</x>
     <y>180</y>
     <width>121</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>空格</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_B">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_0">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_W">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_CapsLock">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>大写</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_R">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_E">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_U">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_N">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>180</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_Q">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>240</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_I">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_S">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_P">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_V">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Key_T">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
